src/backend_connection/error.rs

```diff
@@ -22,6 +22,13 @@ pyo3::create_exception!(
     BackendInvitationOnExistingMember,
     BackendConnectionError
 );
+// TODO: `LoggedCore.new_user_invitation` raise `BackendInvitationOnExistingMember`
```

Comment:
is there something to do ? or is this for the future vxgmichel

>  These are the tales of those times. Forget the power of Rust and Python, for so much has been forgotten, never to be relearned. Forget the promise of progress and understanding, for in the grim dark future there is only PHP4.
> --- future vxgmichel

Reply:
Only a sad copy/paste from there: https://github.com/Scille/parsec-cloud/pull/4921/files/5351af53a47825df933cede7b6b142a84545e41f#diff-b2c3a9e82f595b0292acf758e26ffac1bace35ad68c9c81da51afc29525d7356R18-R19

Reply:
I forgot to rename it accordingly tho